ts/parts/SVGElement.ts

```diff
@@ -240,7 +240,7 @@ declare global {
             ): void;
             public xGetter(key: string): (number|string|null);
             public yGetter(key: string): (number|string|null);
-            public zIndexSetter(value: number, key: string): boolean;
+            public zIndexSetter(value?: number, key?: string): boolean;
```

Comment:
Is that change necessary? What's the point of calling `zIndexSetter` without arguments? Why 3D needs this?

Reply:
In shortcut, you don't need `key` in many use-cases.